apps/site/lib/site_web/controllers/schedule/line_controller.ex

```diff
@@ -44,6 +44,8 @@ defmodule SiteWeb.ScheduleController.LineController do
   end
 
   def assign_schedule_page_data(conn) do
+    service_date = conn.assigns.date_time |> Util.service_date()
```

Comment:
We have `conn.assigns.date` for just this purpose; no need to convert from a datetime 🙂